pkg/reactor/build.go

```diff
@@ -35,15 +35,6 @@ func (r *Reactor) Build(ctx context.Context, documentationStructure []*api.Node)
 	gitInfoShutdownCh := make(chan struct{})
 	loop := true
 
-	defer func() {
-		close(errCh)
-		close(downloadShutdownCh)
-		close(gitInfoShutdownCh)
-		close(documentShutdownCh)
-		close(doneCh)
-		klog.V(2).Infoln("Build finished")
-	}()
-
```

Comment:
Let's keep the explicit channel closing in for now. Plus the "Build finished"  log